prow/pj-on-kind.sh

```diff
@@ -79,7 +79,7 @@ function ensureInstall() {
   # Install kind and set up cluster if not already done.
   if ! command -v kind >/dev/null 2>&1; then
     echo "Installing kind..."
-    GO111MODULE="on" go get sigs.k8s.io/kind@v0.16.0
+    GO111MODULE="on" go install sigs.k8s.io/kind@v0.16.0
```

Comment:
want to update this to v0.17.0? we fixed a few bugs last week.


switching to install will require a higher minimum go version IIRC, so there's also that.

Reply:
> want to update this to v0.17.0? we fixed a few bugs last week.

Thanks, done! But it seems we would have to use Go 1.16 or newer to use this package because `package io/fs` requires Go 16.
I'm getting this from Go 1.15: `go/pkg/mod/github.com/google/safetext@v0.0.0-20220905092116-b49f7bc46da2/yamltemplate/yamltemplate.go:27:2: package io/fs is not in GOROOT (/usr/local/go/src/io/fs)`.

> switching to install will require a higher minimum go version IIRC, so there's also that.

First a disclaimer: I'm not a Go expert :)
It seems this `go get` deprecation is not very backward compatible.  For go version starting from 1.18, `go get` cannot be used to install a package. For versions before that, I tried using the following commands which didn't work
```bash
GO111MODULE="on" go get -d sigs.k8s.io/kind@v0.17.0
GO111MODULE="on" go install sigs.k8s.io/kind
```
Above gives me `cannot find module providing package sigs.k8s.io/kind: working directory is not part of a module`. So, I added a condition check for the current go version. For versions before 18, we use `go get`. For versions starting >= 18, we use `go install`.
I don't know if there are any better ways to do it.